CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ else()
 endif()
 
 # -fPIC is implied on MinGW...
-if(NOT WIN32)
+if((NOT WIN32) AND (NOT MICROCONTROLLER_BUILD))
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

Comment:
Is this still needed, given the experimental nano build didn't work out?

Reply:
Yep, it's still required for any Codec2 builds intended for microcontrollers. I just tried disabling MICROCONTROLLER_BUILD on the Teensy version of freedv_dongle and the test program wouldn't send/receive anything over USB. Reenabling MICROCONTROLLER_BUILD (which disables -fPIC) resolves the problem.

BTW, on Teensy 4.1 (600MHz Cortex M7 w/FPU):

```
$ time ./test -m 700D -t -p /dev/cu.usbmodem106388901 -i ~/devel/codec2/raw/ve9qrp_10s.raw -o tx.raw 

real	0m2.225s
user	0m0.005s
sys	0m0.022s
```